pkg/gatherers/clusterconfig/const.go

```diff
@@ -58,6 +58,14 @@ var (
 	costManagementMetricsConfigResource = schema.GroupVersionResource{
 		Group: "costmanagement-metrics-cfg.openshift.io", Version: "v1beta1", Resource: "costmanagementmetricsconfigs",
 	}
+	operatorGVR = schema.GroupVersionResource{
```

Comment:
I moved the GVR from `pkg/gatherers/clusterconfig/olm_operators.go` to this file to keep some code consistency once all GVRs seem to be declared here.